pyaedt/application/Variables.py

```diff
@@ -1056,7 +1061,7 @@ def set_variable(
                     self._logger.clear_messages()
                     return
             except Exception:
-                pass
+                self._logger.debug("Failed to delete delimiter {}.".format(variable_name))
         else:
             raise Exception("Unhandled input type to the design property or project variable.")  # pragma: no cover
```

Comment:
Rewrite as a complete sentence. "Input type to the design property or project variable is unhandled."
Is there a less "technical way to say this. Plain English is always better.

Reply:
Proposed changes are understandable but it's difficult to pinpoint exactly what went wrong here. I updated the message though.